locales/cs.lua

```diff
@@ -26,3 +39,11 @@ if GetConvar('qb_locale', 'en') == 'cs' then
         fallbackLang = Lang,
     })
 end
+if GetConvar('qb_locale', 'en') == 'cs' then
+    Lang = Locale:new({
+        phrases = Translations,
+        warnOnMissing = true,
+        fallbackLang = Lang,
+    })
+end
+--translate by stepan_valic
```

Comment:
Please remove this comment. The git commit has the author name attached to it.

Reply:
fixed

Reply:
It's not fixed. Your comment is still there.

Reply:
I'll try it tomorrow